test/e2e/helpers.js

```diff
@@ -289,6 +290,97 @@ const completeImportSRPOnboardingFlowWordByWord = async (
   await driver.clickElement('[data-testid="pin-extension-done"]');
 };
 
+const completeCreateNewWalletOnboardingFlow = async (
+  driver,
+  seedPhrase,
+  password,
+) => {
+  // welcome
+  await driver.clickElement('[data-testid="onboarding-create-wallet"]');
+
+  // metrics
+  await driver.clickElement('[data-testid="metametrics-no-thanks"]');
+
+  // create password
+  await driver.fill('[data-testid="create-password-new"]', password);
+  await driver.fill('[data-testid="create-password-confirm"]', password);
+  await driver.clickElement('[data-testid="create-password-terms"]');
+  await driver.clickElement('[data-testid="create-password-wallet"]');
+
+  // secure my wallet
+  await driver.clickElement('[data-testid="secure-wallet-recommended"]');
+
+  // reveal SRP
+  await driver.clickElement('[data-testid="recovery-phrase-reveal"]');
+
+  const revealedSeedPhrase = await driver.findElement(
+    '[data-testid="recovery-phrase-chips"]',
+  );
+
+  let recoveryPhrase = seedPhrase;
+
+  recoveryPhrase = await revealedSeedPhrase.getText();
```

Comment:
Here we pass the parameter `seedPhrase` to` recoveryPhrase` first, then we pass the displayed words to ` recoveryPhrase` again, which makes the first step useless.
Actually I think in this `completeCreateNewWalletOnboardingFlow` function, we don't need the `seedPhrase` parameter.

Reply:
Done